tests/utils.py

```diff
@@ -90,7 +90,7 @@ class EnvironmentVariables:
     LOGIN_FLOW: _LOGIN_FLOW = "infer"
     IDP_CLIENT_ID: str | None = None
     IDP_CLIENT_SECRET: str | None = None
-    TOKEN: str | None = None
+    CDF_TOKEN: str | None = None
```

Comment:
Unrelated change. It is just function I used in debugging. Please forgive me for including these lines in this PR.